.github/workflows/test.yml

```diff
@@ -92,23 +91,19 @@ jobs:
           - "tests/tensor/test_basic.py tests/tensor/test_inplace.py tests/tensor/conv"
           - "tests/tensor/test_blas.py tests/tensor/test_elemwise.py tests/tensor/test_math_scipy.py"
         exclude:
-          - python-version: "3.10"
+          - python-version: "3.11"
             fast-compile: 1
-          - python-version: "3.10"
+          - python-version: "3.11"
             float32: 1
           - fast-compile: 1
             float32: 1
-          - numpy-version: "~=1.26.0"
-            fast-compile: 1
-          - numpy-version: "~=1.26.0"
-            float32: 1
-          - numpy-version: "~=1.26.0"
-            python-version: "3.13"
+          - fast-compile: 1
+          - float32: 1
+          - python-version: "3.13"
```

Comment:
I couldn't comment with suggestion on the phone, but these are excluding all fast-compile/float32/python 3.13 runs. They only made sense in combination with numpy version (we were trying not to blow the CI when we added the numpy variants)
```suggestion
```